src/Tribe/Customizer/Global_Elements.php

```diff
@@ -62,23 +94,50 @@ public function get_css_template( $template ) {
 				.tribe-common .tribe-common-c-btn:focus,
 				.tribe-common .tribe-common-c-btn:hover,
 				.tribe-common a.tribe-common-c-btn:focus,
-				.tribe-common a.tribe-common-c-btn:hover,
-				.tribe-events .tribe-events-calendar-month__day-cell--selected,
-				.tribe-events .tribe-events-calendar-month__mobile-events-icon--event,
-				.tribe-theme-twentytwenty .tribe-events .tribe-events-calendar-month__day-cell--selected {
+				.tribe-common a.tribe-common-c-btn:hover {
+					background-color: ' . $accent_color_hover . ';
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-c-btn:active,
+				.tribe-common a.tribe-common-c-btn:active {
+					background-color: ' . $accent_color_active . ';
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-c-btn:disabled,
+				.tribe-common a.tribe-common-c-btn:disabled {
+					background-color: ' . $accent_color_background . ';
+				}
+			';
+
+			$template .= '
+				.tribe-theme-twentytwenty .tribe-common .tribe-common-c-btn {
 					background-color: <%= global_elements.accent_color %>;
 				}
 			';
 
 			$template .= '
-				.tribe-theme-twentytwenty .tribe-common .tribe-common-c-btn:hover,
-				.tribe-theme-twentytwenty .tribe-common .tribe-common-c-btn:focus a,
 				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn:hover,
-				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn:focus a {
-					background: rgba( ' . $accent_css_rgb . ', 0.8);
+				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn:focus,
+				.tribe-theme-twentytwenty .tribe-common .tribe-common-c-btn:hover,
+				.tribe-theme-twentytwenty .tribe-common .tribe-common-c-btn:focus {
+					background-color: ' . $accent_color_hover . ';
 				}
 			';
 
+			// overrides for common components/full/_loader.pcss
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 85e41ff53d1ffd6d18649d3db70ca9b5</sub>